### PR TITLE
Call automation live tests

### PR DESCRIPTION
**Pull request by juntuchen-msft**
- Added live tests scenarios for call automation service

### CODE REVIEW

**Comment by yyw-msft** [comment]
why is there "[1]" at the end of each session record json file name? is this how the java ones r set up?

**Comment by juntuchen-msft** [comment]
Yes, this is expected, and it is auto generated by maven test.

File: sdk/communication/azure-communication-callautomation/src/test/resources/session-records/CallAutomationAsyncClientLiveTests.createPSTNCallAndHangupTest[1].json [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      "Content-Type" : "application/json"
+    },
+    "Response" : {
+      "X-Cache" : "CONFIG_NOCACHE",
+      "content-length" : "576",
+      "retry-after" : "0",
+      "X-Azure-Ref" : "0/u8tYwAAAADGmnT7zsrtS6dyvg/qszGhQ0hHRURHRTE1MDYAOWZjN2I1MTktYThjYy00Zjg5LTkzNWUtYzkxNDhhZTA5ZTgx",
+      "StatusCode" : "201",
+      "X-Microsoft-Skype-Chain-ID" : "99b9d62d-3013-47d2-8dd8-73fb745390b7",
+      "Body" : "{\"callConnectionId\":\"811f6a00-d2c9-4139-86ff-e46cbdde30d9\",\"source\":{\"callerId\":{\"value\":\"+18337597849\"},\"identifier\":{\"rawId\":\"8:acs:816df1ca-971b-44d7-b8b1-8fba90748500_00000014-0b53-1f54-99bf-a43a0d009143\",\"kind\":\"communicationUser\",\"communicationUser\":{\"id\":\"8:acs:816df1ca-971b-44d7-b8b1-8fba90748500_00000014-0b53-1f54-99bf-a43a0d009143\"}}},\"targets\":[{\"rawId\":\"4:+16476282217\",\"kind\":\"phoneNumber\",\"phoneNumber\":{\"value\":\"+16476282217\"}}],\"callConnectionState\":\"connecting\",\"callbackUri\":\"https://localhost\",\"mediaSubscriptionId\":\"72e11d73-b9af-4e40-8993-be50c260ff09\"}",
```
Comment (yyw-msft) [inline sdk/communication/azure-communication-callautomation/src/test/resources/session-records/CallAutomationAsyncClientLiveTests.createPSTNCallAndHangupTest[1].json:42]: can the phone numbers and acs ids be sanitized?

**Review by cochi2 — APPROVED** [review]
Tests seem to be good.

File: sdk/communication/azure-communication-callautomation/src/test/resources/session-records/CallAutomationAsyncClientLiveTests.createPSTNCallAndHangupTest[1].json (hunk 1 above)
Comment (juntuchen-msft) [reply sdk/communication/azure-communication-callautomation/src/test/resources/session-records/CallAutomationAsyncClientLiveTests.createPSTNCallAndHangupTest[1].json]: Updated. I have marked them as REDACTED.

**Review by yyw-msft — APPROVED** [review]

**Pull request merged**